core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -1186,6 +1185,9 @@ class GroupCoordinator(val brokerId: Int,
           for (member <- group.allMemberMetadata) {
             val joinResult = JoinGroupResult(
               members = if (group.isLeader(member.memberId)) {
+                info(s"Group ${group.groupId} replies the leader with " +
```

Comment:
Do we need to print this for every member? That is O(n^2). Keep in mind that groups in the thousands are not unheard of. I think it would be better if we just ensured that we log the id of every member entering and leaving the group.